### DISCUSSION TITLE
Is there any plan for enum member description

### DISCUSSION THREAD

**Discussion by hyprocritor** [graphql-nexus/nexus-prisma#448] [General]
I just found that the enum has the description, but seems that it does not support the description for enums values.
```
/// This is an enum description, this has been supported
enum EnumStaff{
  /// this is a description for the enum value1
  Value
  /// this is a description for the enum value2, but seems not available yet
  Value1
}
```

**Comment by hyprocritor** [comment]
Just found that does not support the keyword `type` either. 
Here is the detail about the `type` type: https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#composite-type-unique-constraints

**Reply by rostislav-simonik** [reply]
@czk1997 do you feel you could help with this ?
